src/app/content/types.ts

```diff
@@ -107,3 +107,5 @@ export interface ArchivePage {
 }
 
 export type ArchiveContent = ArchivePage | ArchiveBook;
+
+export type BookSections = Map<string, LinkedArchiveTree | LinkedArchiveTreeSection>;
```

Comment:
the way you're using this its a highlighting specific thing based on the filter options available for highlighting, i'd keep it in the highlighting types